src/magic/data/RomanToInt.java

```diff
@@ -0,0 +1,18 @@
+package magic.data;
+
+public class RomanToInt {
+
+    public static int convert(String num) {
+        if (num == null) {
```

Comment:
null case should never happen, since a chapter indicator must be there, please remove this.